rpc/gnmi/gnmi-specification.md

```diff
@@ -453,6 +453,35 @@ update: <
 >
 ```
 
+For `/a/b[name=b1]`:
+
+```
+update: <
+  path: <
+    elem: <
+      name: "a"
+    >
+    elem: <
+      name: "b"
+      key: <
+        key: "name"
+        value: "b1"
+      >
+    >
+  >
+  val: <
+    json_ietf_val: `{ "b": {
+                        "name": "b1",
+                        "c": {
+                          "d": "AStringValue",
+                          "e": 10042
+                        }
+                   }
+            }`
```

Comment:
Is this correct? should the "b"'s level be taken out?

Reply:
Yes, I'll fix it: https://github.com/openconfig/reference/issues/139#issuecomment-840829882